cumulusci/tasks/release_notes/parser.py

```diff
@@ -295,9 +295,12 @@ def parse(self, change_note):
 
     def render(self, existing_content=""):
         version_id = self.release_notes_generator.version_id
+        release_info = self.release_notes_generator.release_info
+        trial_info = self.release_notes_generator.trial_info
+
         if version_id:
             version_id = urllib.parse.quote_plus(version_id)
-            return "\r\n".join(
+            existing_content += "\r\n".join(
```

Comment:
`existing_content` is the existing text of this section if the task is run against a release that already has release notes with this section. The way you've coded this with +=, it will add a new duplicate copy of this information each time the task is run. Instead, we should:
1. re-regenerate the entire section content if any of version_id, sandbox_date, production_date, or trial_info were specified.
2. otherwise return existing_content untouched (so it's possible to run github_release_notes manually without specifying any of the options, but not wipe out the existing content of this section)